shared/app/index.native.tsx

```diff
@@ -18,9 +18,18 @@ module.hot?.accept(() => {
   console.log('accepted update in shared/index.native')
 })
 
-const NativeEventsToRedux = () => {
+const NativeEventsToRedux = (p: {setDarkMode: (d: boolean) => void}) => {
+  const {setDarkMode} = p
   const dispatch = useDispatch()
-  const appStateRef = React.useRef('unknown')
+  const appStateRef = React.useRef('active')
```

Comment:
while debugging this can be left as unknown (like on a refresh) which will cause you to ignore update so lets just init as active as it should be the case